src/chat/chat_tests.rs

```diff
@@ -2627,6 +2627,121 @@ async fn test_can_send_group() -> Result<()> {
     Ok(())
 }
 
+/// Tests that in a broadcast channel,
+/// the recipients can't see the identity of their fellow recipients.
+#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+async fn test_broadcast_members_cant_see_each_other() -> Result<()> {
+    fn contains(parsed: &MimeMessage, s: &str) -> bool {
```

Comment:
Maybe add this as a function to `test_utils` or even `MimeMessage` (under `#[cfg(test)]`)? It'll be useful e.g. for tests on header protection